rpcserver.go

```diff
@@ -3180,6 +3181,25 @@ func handleGetTxOut(_ context.Context, s *rpcServer, cmd interface{}) (interface
 	return txOutReply, nil
 }
 
+// handleGetTxOutSetInfo returns statistics on the current unspent transaction output set.
+func handleGetTxOutSetInfo(_ context.Context, s *rpcServer, cmd interface{}) (interface{}, error) {
+	best := s.cfg.Chain.BestSnapshot()
+	stats, err := s.cfg.Chain.FetchUtxoStats()
+	if err != nil {
+		return nil, err
+	}
+
+	return types.GetTxOutSetInfoResult{
+		Height:         best.Height,
+		BestBlock:      best.Hash.String(),
+		Transactions:   stats.Transactions,
+		TxOuts:         stats.Utxos,
+		DiskSize:       stats.Size / 1000,
```

Comment:
This matches what the help says, but what do you think about making it bytes instead?  I generally prefer the RPCs to be in the smallest units possible so the caller can choose how to display it.  Converting it to `kB` here loses precision the caller can't undo, whereas the caller can easily convert bytes to `kB` if it so desires.